pkg/sriov/sriov.go

```diff
@@ -231,7 +210,10 @@ func (s *sriovManager) ApplyVFConfig(conf *sriovtypes.NetConf) error {
 			return fmt.Errorf("failed to parse MAC address %s: %v", conf.MAC, err)
 		}
 
-		if err = s.nLink.LinkSetVfHardwareAddr(pfLink, conf.VFID, hwaddr); err != nil {
+		err = utils.Retry(5, 200*time.Millisecond, func() error {
```

Comment:
Thats a new one since i last reviewed (i think .. :) )

I assume its to align with how we do things in ResetVFConfig.

its unclear why we are retrying here... (and also after removing the comment at L350 also in resetVFConfig)

either:
1. define a wrapper func "SetVFHardwareAddress" in utils which will do this and add comment to that method.
2. add comments here and below to explain why its being retired.

Reply:
it was to be equal with the other functions base on this comment https://github.com/k8snetworkplumbingwg/sriov-cni/pull/239#issuecomment-1561650221

1. I create the function and also add unit tests for that new function
2. I add a small comment